docs/running/interaction/hdf_writer.ipynb

```diff
@@ -67,9 +63,7 @@
   {
    "cell_type": "code",
    "execution_count": 3,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
```

Comment:
Does this automatically becomes empty in Python 3 or did you do it for any specific purpose?

Reply:
I didn't change it, let me look into it

Reply:
Actually this automatically becomes empty when for python 3

Reply:
So you practically ran these both notebooks in python3 (latest tardis) environment?

Reply:
Yes